api/Window.json

```diff
@@ -9042,11 +9044,11 @@
               "version_added": false
             },
             "firefox": {
-              "notes": "From Firefox 78 <code>AddSearchProvider()</code> does nothing, as the specification requires.",
-              "version_added": true
+              "version_added": "1",
+              "notes": "From Firefox 78 <code>AddSearchProvider()</code> does nothing, as the specification requires."
```

Comment:
Looks like a big change in the diff, but it's just fixing the order, note is unchanged.

Reply:
This note is really questionable. We should probably break this out into subfeatures. Opened https://github.com/mdn/browser-compat-data/issues/7548